azurelinuxagent/common/exception.py

```diff
@@ -281,3 +282,15 @@ class ExtensionErrorCodes(object):
 
     def __init__(self):
         pass
+
+
+class GoalStateAggregateStatusCodes(object):
```

Comment:
ExtensionErrorCodes shouldn't be here either, but then again the whole status thing shouldn't be in exthandler.py so I guess it doesn't really matter since we would need a good refactoring to separate this code in a modular way.

Reply:
yeah exactly. We can tackle this in the new agent I suppose